grid/app/main/events/dfl/model_events.py

```diff
@@ -0,0 +1,132 @@
+import json
```

Comment:
You should put a blank line between each group of imports.
You should group the third party imports together